README.md

```diff
@@ -33,7 +33,19 @@ To test the project, you need first to install the server and get it running, th
 To install and start the server:
 
 1. Install Docker.
-   - [On Windows](https://docs.docker.com/docker-for-windows/install/).
+   - [On Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/).
+   - [On Windows 10 Home and below](https://github.com/docker/toolbox/releases).
```

Comment:
I would move that to a dedicated heading after the main download list, as there are multiple gotchas for Windows home.

E.g.

> ### Setting up docker on Windows 10 Home (and below)
>
> You need to download the [docker toolbox](https://github.com/docker/toolbox/releases)
> ... [add instructions here]

Reply:
noted!